<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>BiliBili直播推流码获取</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="home_right_button" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>100</width>
     <height>260</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> background: rgba(255, 255, 255,0.2);
 backdrop-filter: blur(10px);
 border-radius: 12px;
 border: 1px solid rgba(0, 0, 0, 0.1);
 box-shadow: 0 4px 6px rgba(0, 0, 0, 0.1), 0 1px 3px rgba(0, 0, 0, 0.08);
 width: 100%;
 box-sizing: border-box;</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QPushButton" name="live_start">
      <property name="text">
       <string>开始直播</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="live_stop">
      <property name="text">
       <string>停止直播</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="get_stram">
      <property name="text">
       <string>获取推流码</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="setting">
      <property name="text">
       <string>设置</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="change_mode">
      <property name="text">
       <string>切换模式</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="login">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
